src/styles.css

```diff
@@ -102,3 +107,69 @@ body {
   width: 50px;
   height: 50px;
 }
+.organism-description {
+  font-weight: bold;
+}
+.new-game-popup-container {
+  position: fixed;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%) scale(1);
+  transition: 200ms ease-in-out;
+  border: 1px solid black;
+  border-radius: 10px;
+  z-index: 10;
+  background-color: white;
+  width: 650px;
+  max-width: 80%;
+}
+.new-game-popup-container.inactive {
+  transform: translate(-50%, -50%) scale(0);
+}
+.new-game-header {
+  display: flex;
+  justify-content: center;
+  padding: 10px 15px;
+  border-bottom: 1px solid black;
+  font-size: 20px;
+  font-weight: bold;
+}
+.new-game-button {
+  background-color: #4CAF50;
+  border: none;
+  color: white;
+  padding: 15px 32px;
+  text-align: center;
+  font-size: 16px;
+  cursor: pointer;
+}
+.new-game-button-container {
+  display: flex;
+  justify-content: center;
+  margin-bottom: 5px;
+}
+.text-style {
+  margin: 5px 5px 5px 5px;
+  font-weight: bold;
+}
+.control-buttons {
+  position: fixed;
+  top: 50%;
+  left: 85%;
+  transform: translate(-50%, -50%) scale(0);
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  width: 170px;
+  height: 170px;
+}
+.control-buttons.active {
+  transform: translate(-50%, -50%) scale(1);
```

Comment:
Avoid using position: fixed and position: absolute, because it can complicate your layout a lot.
Centering it on the screen can be achieved with flexbox using justify-content and align-items.